test-utils/src/dev/falcon.rs

```diff
@@ -4,10 +4,62 @@
 
 //! Example test to run in a VM.
 
-#[cfg(all(test, target_os = "illumos"))]
 mod test {
+    use super::*;
+    use crate::dev::test_setup_log;
     use anyhow::anyhow;
     use gethostname::gethostname;
+    use libfalcon::unit::gb;
+
+    #[ignore]
+    #[tokio::test]
+    async fn run_launch_in_vm() {
+        let args: Vec<_> = std::env::args().collect();
+        let logctx = test_setup_log("launch");
+        let log = logctx.log.clone();
+        let runner_name = "launchpad_mcduck";
+        let node_name = "launchpad_mcduck_test_vm";
+        let mut test = super::FalconTest::new(runner_name, &log).unwrap();
+        eprintln!("args = {:?}", args);
+        // Create a falcon node (VM)
+        let vm = test.runner().node(node_name, "helios-2.0", 1, gb(2));
+
+        // Create a place(cargo-bay) to store data to be mounted into the VM
+        let cargo_bay = tempdir().unwrap();
+        info!(log, "Set cargo-bay to {} on host machine", cargo_bay.path());
+        let test_name = "launch";
+        let source_test_path = &args[0];
```

Comment:
This is a hack to easily find the source binary. It works only if the test to run in the VM is part of the same test binary generated by cargo.